pom.xml

```diff
@@ -12,29 +12,9 @@
   <parent>
     <groupId>org.quattor.maven</groupId>
     <artifactId>build-profile</artifactId>
-    <version>1.38</version>
+    <version>1.40</version>
   </parent>
 
-  <scm>
-    <connection>scm:git:git://github.com/quattor/CCM.git</connection>
-    <developerConnection>scm:git:git@github.com:quattor/CCM.git</developerConnection>
-    <url>https://github.com/piojo/CCM</url>
-  </scm>
```

Comment:
Whoops :fearful:, we can't release without the `<scm>` block, I'll open a PR to restore it!